docs/user/supported_devices.rst

```diff
@@ -206,6 +206,10 @@ ath79-generic
 
   - Archer C6 (v2)
 
+* Ubiquity
```

Comment:
The company is called Ubiquiti with an "i" at the end.

Reply:
The device needs to be flagged as broken, so it won't appear on the supported devices list at all.